src/gui/owncloudsetupwizard.cpp

```diff
@@ -124,12 +125,21 @@ void OwncloudSetupWizard::slotDetermineAuthType(const QString &urlString)
         url.setScheme("https");
     }
     AccountPtr account = _ocWizard->account();
+
+    // Detect a Sandstorm web key and derive the credentials from it.
+    if (url.hasFragment() && url.path() == "") {
+      account->setCredentials(new HttpCredentialsGui("sandstorm", url.fragment(), NULL, NULL));
+      account->setCredentialSetting("user", "sandstorm");
+      account->setCredentialSetting("password", url.fragment());
+    } else {
+      // Set fake credentials beforfe we check what credential it actually is.
```

Comment:
s/beforfe/before
Might as well fix the typo that was in the original comment text :)